Detectors/FIT/FV0/simulation/src/Digitizer.cxx

```diff
@@ -209,13 +223,21 @@ Float_t Digitizer::SimulateTimeCfd(Int_t channel)
   return timeCfd;
 }
 
+Double_t Digitizer::PmtResponse(Double_t* x, Double_t*)
+{
+  return Digitizer::PmtResponse(x[0]);
+}
 //_______________________________________________________________________
-Float_t Digitizer::PmtResponse(Float_t x)
+Double_t Digitizer::PmtResponse(Double_t x)
```

Comment:
@hcab14 I also needed to change this to doubles, because otherwise TF1 was generating warnings: `Error 18 in qags.c at 548 : cannot reach tolerance because of roundoff error`
Another way would be to change the ROOTs tolerance, but I have little experience with that and think it would then work globally.